src/app/api/onramp-coinbase/route.ts

```diff
@@ -52,7 +52,7 @@ export async function POST(req: Request) {
     }
 
     const jwt = sign(payload, key_secret, signOptions)
-
+    // Have to hardcode base because it doesn't support base sepolia
```

Comment:
I want to understand better the impact of this solution, does that mean that if I try to onramp on development it will onramp base mainnet funds or how does it work? it seems a bit confusing.